sentry-graphql/src/main/java/io/sentry/graphql/SentryInstrumentation.java

```diff
@@ -1,49 +1,73 @@
 package io.sentry.graphql;
 
+import graphql.ErrorClassification;
 import graphql.ExecutionResult;
+import graphql.GraphQLContext;
+import graphql.GraphQLError;
+import graphql.execution.ExecutionContext;
+import graphql.execution.ExecutionStepInfo;
 import graphql.execution.instrumentation.InstrumentationContext;
 import graphql.execution.instrumentation.InstrumentationState;
 import graphql.execution.instrumentation.SimpleInstrumentation;
+import graphql.execution.instrumentation.parameters.InstrumentationExecuteOperationParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationExecutionParameters;
 import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import graphql.execution.instrumentation.parameters.InstrumentationFieldParameters;
+import graphql.language.OperationDefinition;
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 import graphql.schema.GraphQLNonNull;
 import graphql.schema.GraphQLObjectType;
 import graphql.schema.GraphQLOutputType;
-import io.sentry.HubAdapter;
+import io.sentry.Breadcrumb;
 import io.sentry.IHub;
 import io.sentry.ISpan;
+import io.sentry.NoOpHub;
+import io.sentry.Sentry;
 import io.sentry.SentryIntegrationPackageStorage;
 import io.sentry.SpanStatus;
-import io.sentry.util.Objects;
+import io.sentry.util.StringUtils;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CopyOnWriteArrayList;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 public final class SentryInstrumentation extends SimpleInstrumentation {
-  private final @NotNull IHub hub;
+
+  private static final @NotNull List<String> ERROR_TYPES_HANDLED_BY_DATA_FETCHERS =
+      Arrays.asList("INTERNAL", "INTERNAL_ERROR");
   private final @Nullable BeforeSpanCallback beforeSpan;
+  private final @NotNull SentrySubscriptionHandler subscriptionHandler;
+
+  private final @NotNull ExceptionReporter exceptionReporter;
+
+  // TODO ctor that takes a hub
```

Comment:
Are we going to address this?

Reply:
yeah, there's comments on the PR to address